_sass/vrsite/assignments.scss

```diff
@@ -1,9 +1,9 @@
-.table-header {
+.assignments > .table-header {
```

Comment:
any specific reason why this is the "direct child" selector? other ones are just the generic descendant selector

Reply:
My bad, that actually broke the CSS. Fixed.